cmd/diff-main.go

```diff
@@ -34,25 +34,21 @@ var (
 
 // Compute differences between two files or folders.
 var diffCmd = cli.Command{
-	Name:        "diff",
-	Usage:       "Show differences between two folders or buckets.",
-	Description: "Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. Objects of same name and size, but differ in contents are not noticed.",
-	Action:      mainDiff,
-	Before:      setGlobalsFromContext,
-	Flags:       append(diffFlags, globalFlags...),
+	Name:   "diff",
+	Usage:  "List objects with size difference or missing between two folders or buckets.",
+	Action: mainDiff,
+	Before: setGlobalsFromContext,
+	Flags:  append(diffFlags, globalFlags...),
 	CustomHelpTemplate: `NAME:
   {{.HelpName}} - {{.Usage}}
-
 USAGE:
   {{.HelpName}} [FLAGS] FIRST SECOND
 
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 DESCRIPTION:
-  {{.Description}}
-
-OUTPUT:
+  Diff only lists missing objects or objects with size differences. It *DOES NOT* compare contents. i.e. objects of same name and size, but differing in content are not noticed.
```

Comment:
We can take this verbatim description from the `diffCmd` initializer and keep the help template simpler and clean.